lib/components/canvas/index.ts

```diff
@@ -165,15 +166,13 @@ export class CanvasSink extends Sink {
 
           // Initialize first timestamp and clockrate
           firstTimestamp = 0
-          const jpegMedia = msg.sdp.media.find(
-            (media): media is VideoMedia => {
-              return (
-                media.type === 'video' &&
-                media.rtpmap !== undefined &&
-                media.rtpmap.encodingName === 'JPEG'
-              )
-            },
-          )
+          const jpegMedia = msg.sdp.media.find((media): media is VideoMedia => {
```

Comment:
There must be still something odd with your prettier setup, this line should be split.

Reply:
Reinstalled everyting using `--frozen-lockfile` and result is the same. 
`
reijo@rmackbook axis (safari_mjpeg) $ yarn prettier lib/components/canvas/index.ts | grep "const jpegMedia" -A 2 -B 2
          // Initialize first timestamp and clockrate
          firstTimestamp = 0
          const jpegMedia = msg.sdp.media.find((media): media is VideoMedia => {
            return (
              media.type === 'video' &&
`
Can you please push a commit how ever you like it ?

Reply:
I'll have a look ASAP.

Reply:
hmm, seemed to be wrong on master, so I'll just merge this. I'll add a test on master to check and reformat if necessary.